testing/src/cypress/e2e/04-api-institution/01-instition-dashboard.cy.ts

```diff
@@ -1,23 +1,27 @@
-import InstitutionHelperActions from "../02-institution/common-helper-functions.cy";
-
+import InstitutionHelperActions from "../../custom-command/institution/common-helper-functions.cy";
+import Authorization from "../../custom-command/common/authorization";
 const institutionHelperActions = new InstitutionHelperActions();
 
 const INSTITUTION_DETAILS_SINGLE_LOCATION =
   institutionHelperActions.getInstitutionDetailsSingleLocation();
 const USER_DETAILS_SINGLE_LOCATION =
   institutionHelperActions.getUserDetailsSingleLocation();
 const API_URL = institutionHelperActions.getApiUrlForTest();
-const TOKEN = institutionHelperActions.getToken();
 
-describe("Validate Institution apis - Institution with single location", () => {
-  it("Get User status", () => {
+describe("[Institution Dashboard APIs] - Institution with single location", () => {
+  let token: any;
```

Comment:
string?

Reply:
updated

Reply:
not sure, if it's my gut issue. i could see the changes